command-acceptor/app/controllers/implementations/Mediator.py

```diff
@@ -3,29 +3,48 @@
 from app.controllers.implementations.CommandFactory import CommandFactory
 from model.AcceptedMessage import AcceptedMessage
 from model.Command import Command
+from model.ErrorMessage import ErrorMessage
 from threading import Event
+from app.controllers.implementations.ProcessorQueuePusher import ProcessorQueuePusher
+from app.controllers.implementations.ErrorQueuePusher import ErrorQueuePusher
+from model.SentMessageContainer import SentMessageContainer
+from model.ErrorMessageContainer import ErrorMessageContainer
 
 
 class Mediator(interfaces.Mediator):
-    def __init__(self, event: Event, message_holder: MessageHolder)-> None:
+    def __init__(self, event: Event, message_holder: MessageHolder, processor_queue_pusher: ProcessorQueuePusher,
+                 error_queue_pusher: ErrorQueuePusher) -> None:
         self.busy: bool = False
```

Comment:
busy is coming from the fact that your object is doing some work, but in your case this flag is for instructing of your object if it has to work or not. I would rather rename it to 'active'

Reply:
Have changed to active